views/firstLogin/sections/parent_data.hbs

```diff
@@ -1,24 +1,24 @@
 <section data-panel="section-{{section-nr}}" {{#if submit-page}}class="submit-page"{{/if}}>
-  <h2 class="h4">Ihre Daten</h2>
-  <p>Für Schüler unter {{CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS}} Jahren bedarf ein Erziehungsberechtigter in die Nutzung der {{@root.theme.short_title}} einwilligen.</p>
-  <p>Dazu benötigen wir folgende Angaben von Ihnen:</p>
+  <h2 class="h4">{{$t "firstLogin.parent.heading.yourData" }}</h2>
+  <p>{{$t "firstLogin.parent.global.text.forStudentsUnder" }} {{CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS}} {{$t "firstLogin.parent.text.aParentOrLegalGuardianMystUse" }} {{@root.theme.short_title}} {{$t "firstLogin.parent.text.toConsent" }}</p>
+  <p>{{$t "firstLogin.parent.text.forThisWeNeedTheFollowingInformationFromYou" }}</p>
   <div class="row">
     <div class="form-group col-md-6 col-sm-12">
-      <label>Vorname: *</label>
+      <label>{{$t "firstLogin.parent.global.label.firstName" }} *</label>
       <input class="form-control linked" name="parent_firstName" type="text" placeholder="Ihren Vornamen hier eingeben" required/>
```

Comment:
placeholder not translated

Reply:
Its Translated actually.

Reply:
the placeolder Ihren Vornamen hier eingeben is not translated